src/chat/ChatScreen.tsx

```diff
@@ -63,6 +63,8 @@ interface ChatScreenProps extends GiftedChatProps {
   timeoutSendNotify?: number;
   enableTyping?: boolean;
   typingTimeoutSeconds?: number;
+  enableSeenMessage?: boolean;
+  viewUnReadCustom?: (hasUnread: boolean) => JSX.Element;
```

Comment:
I think the name of this prop should be `customUnReadView` or `customViewUnRead`